papyrus/protocol.py

```diff
@@ -325,6 +333,8 @@ def create(self, request):
             if obj is None:
                 obj = self.mapped_class(feature)
                 create = True
+            elif self.before_insert is not None:
```

Comment:
`before_insert` should not be in `elif` after `if obj is None`, or this is a `before_update`

Reply:
Oups, thanks :-)